db/002-hypertable.sql

```diff
@@ -11,11 +11,11 @@ BEGIN
 
     -- Only execute this script if its migration ID is next after the last successful migration ID
     IF migrationId - lastSuccessfulMigrationId = 1 THEN
-        CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
-        CREATE TABLE xethru (LIKE xethru_sensordata INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES);
-        SELECT create_hypertable('xethru', 'published_at');
-        SELECT set_chunk_time_interval('xethru', interval '1 hour');
-        DROP TABLE xethru_sensordata;
+        -- CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
```

Comment:
I'm actually not sure what I want re: this script. It creates something which gets deleted later so I wonder how valuable it is. I commented the contents out because there was an error in it and I was trying to narrow down problems in the build.